### PR TITLE
lxccontainer: fix dereferenced pointer

### PR DESCRIPTION
**Pull request by 2xsec**
Hello,

dereferenced pointer issues are fixed.

Thanks.

Signed-off-by: 2xsec <dh48.jeong@samsung.com>

### CODE REVIEW

**Comment by brauner** [comment]
jenkins: test this please

File: src/lxc/lxccontainer.c [hunk 1]
```diff
@@ -583,6 +583,9 @@ static int do_lxcapi_console_log(struct lxc_container *c, struct lxc_console_log
 {
 	int ret;
 
+	if (!c)
+		return -1;
```
Comment (Blub) [inline src/lxc/lxccontainer.c]: Thanks for the PR.
Seeing how the rest of this function is returning negative errno codes this should probably return `-EINVAL`.

Comment (2xsec) [reply src/lxc/lxccontainer.c]: @Blub It's better. Thanks for your review.:)

Comment (brauner) [reply src/lxc/lxccontainer.c]: Good point. Set errno or negative return in API calls?

**Comment by brauner** [comment]
Thanks! Sorry for the delay I'm currently travelling.

**Comment by 2xsec** [comment]
@brauner It's OK. Have a nice trip :)

**Pull request merged**